llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2098,23 +2099,30 @@ class VPReductionPHIRecipe : public VPHeaderPHIRecipe {
   /// The phi is part of an ordered reduction. Requires IsInLoop to be true.
   bool IsOrdered;
 
+  /// The scaling difference between the size of the output of the entire
+  /// reduction and the size of the inputs When expanding the reduction PHI, the
```

Comment:
```suggestion
  /// reduction and the size of the inputs when expanding the reduction PHI, the
```

Reply:
The formatter messed this comment up. Fixed now.